tests/instrumentation/transactions_store_tests.py

```diff
@@ -352,13 +352,13 @@ def test_set_transaction_name(elasticapm_client):
 
     elasticapm_client.begin_transaction("test")
 
-    elasticapm.set_transaction_name("another_name")
+    elasticapm.set_transaction_name(["another_name"])
```

Comment:
I'd prefer if string coercing would be its own test (or use `pytest.mark.parametrize`)

Reply:
Done.